tests/core/test_basics.py

```diff
@@ -139,6 +139,40 @@ def expected_response_type() -> Type[DummyResponse]:
     assert stream_counter == 2
 
 
+@pytest.mark.asyncio
+async def test_stream_can_get_cancelled(endpoint: Endpoint) -> None:
+    stream_counter = 0
+
+    async_generator = endpoint.stream(DummyRequest)
+
+    async def stream_response() -> None:
+        async for event in async_generator:
+            # Accessing `ev.property_of_dummy_request` here allows us to validate
+            # mypy has the type information we think it has. We run mypy on the tests.
+            print(event.property_of_dummy_request)
+            nonlocal stream_counter
+            stream_counter += 1
+            await asyncio.sleep(0.5)
+
+    async def cancel_soon() -> None:
+        while True:
+            await asyncio.sleep(0.01)
+            if stream_counter == 2:
+                await async_generator.aclose()
+
+    asyncio.ensure_future(stream_response())
+    asyncio.ensure_future(cancel_soon())
+
+    # we broadcast one more item than what we consume and test for that
```

Comment:
the counts seem to not match up with this comment since 50 is not "one more than `2`"

Reply:
Ah copy and paste bug :)